packages/core/src/routes/interaction/mfa.ts

```diff
@@ -103,7 +105,7 @@ export default function mfaRoutes<T extends IRouterParamContext>(
           code: 'session.mfa.mfa_sign_in_only',
         })
       );
-      createLog(`Interaction.${interactionStorage.event}.Mfa.Totp.Submit`);
+      createLog(`Interaction.${interactionStorage.event}.Mfa.${verifyMfaPayloadGuard.type}.Submit`);
```

Comment:
nit: do we have any test could guard this kind of bugs?

Reply:
no, we don't have any tests for "log".